src/components/imageuploader/imageuploader.template.html

```diff
@@ -22,7 +22,7 @@ <h2 style="margin:0;">${HeaderAddUpdateImage}</h2>
                 <div class="imageEditor-dropZone fieldDescription">
                     <div>${LabelDropImageHere}</div>
                     <output id="imageOutput" class="flex align-items-center justify-content-center" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;"></output>
-                    <input type="file" accept="image/png,image/x-png,image/jpeg" id="uploadImage" name="uploadImage" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;opacity:0;" />
+                    <input type="file" accept="image/png,image/x-png,image/jpeg,image/webp" id="uploadImage" name="uploadImage" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;opacity:0;" />
```

Comment:
```suggestion
                    <input type="file" accept="image/*" id="uploadImage" name="uploadImage" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;opacity:0;" />
```

Reply:
@XVicarious are you able to commit the suggestion from @dkanada?